libbeat/docs/outputconfig.asciidoc

```diff
@@ -198,7 +198,7 @@ operation doesn't succeed after
 ===== bulk_max_size
 
 The maximum number of events to bulk in a single Elasticsearch bulk API index request.
-The default is 50.
+The default is 200.
```

Comment:
Whoops. Monica mentioned that you changed the default, and since the setting only appears in one place, I assumed I was changing the correct setting. :-) I'll revert this value to 50 and wait for you to submit a description.